packages/react-shopify-app-route-propagator/README.md

```diff
@@ -100,3 +100,13 @@ export default function() {
   );
 }
 ```
+
+### Troubleshooting
```

Comment:
Why is this a 3rd level heading? Can we :fire: this and just nest the level 4 heading under the `withRouter` section above?